src/data/analyse_en.json

```diff
@@ -78,10 +78,10 @@
             "totalKey": ["app_downloads_cumulated"]
         },
         {
-            "title": "Transmitted and registered test results",
+            "title": "Provided test results",
```

Comment:
I suggest changing the word order

"Test results provided"

which sounds more English.